{{cookiecutter.app_name}}/lib/api/docs/routers/router.go

```diff
@@ -0,0 +1,14 @@
+package routers
+
+import (
+	"github.com/nimblehq/{{cookiecutter.app_name}}/lib/api/docs/controllers"
+
+	"github.com/gin-gonic/gin"
+)
+
+func ComebineRoutes(engine *gin.Engine) {
```

Comment:
is it typo?
```suggestion
func CombineRoutes(engine *gin.Engine) {
```

and combine routes name itself sounds strange for me 🤔

Reply:
Fixed in https://github.com/nimblehq/gin-templates/pull/107/commits/05db79f40c33bb4f2b77d19e3a98d95e534748bd

That's typo. And I follow the existing naming for that function. Any suggestion from you?

Reply:
Just `Routes` will not work?

Reply:
I think it's better to create an issue for this naming, it's better to have feedback from others.

Reply:
I created an issue for this

Reply:
thanks